src/main/java/uk/gov/justice/digital/delius/transformers/OrderManagerTransformer.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.delius.transformers;
+
+import uk.gov.justice.digital.delius.data.api.OrderManager;
+
+import java.util.Optional;
+
+public class OrderManagerTransformer {
+
+    public static OrderManager orderManagerOf(uk.gov.justice.digital.delius.jpa.standard.entity.OrderManager entity){
+        return OrderManager.builder().
+            name(Optional.ofNullable(entity.getStaff().getForename()).orElse("").
+                concat(" "+Optional.ofNullable(entity.getStaff().getSurname()).orElse(""))).
```

Comment:
If the surname is empty here, then we'll end up with an extra space at the end of the name.  Maybe we should return the forenames and surname separately to keep it simple?

Also there is a `forename2` field on the `Staff` entity (like middle name), so if you stick with joining the names then that should be included too. Maybe something like this would be better?
```suggestion
            name(Stream.of(entity.getStaff().getForename(), entity.getStaff().getForename2(), entity.getStaff().getSurname())
				.filter(StringUtils::hasLength)
				.collect(joining(delimiter))).
```
(which would match the method used in the user management tool: https://github.com/ministryofjustice/ndelius-um/blob/main/src/main/java/uk/co/bconline/ndelius/util/NameUtils.java)

Reply:
The requirement is just for name. I think should never be null, they are both defined as not null in the database. I have used Optional.ofNUllable just for test purpose, because in 58 tests  are passed as null. The delimiter idea is good and I guess should be an empty space . I think Name utils needs to be reviewed ;) eheh.